### ISSUE TITLE
Add generic INSERT...RETURNING support for postgres

### ISSUE THREAD

**Issue by kbrowder** [scalikejdbc/scalikejdbc#559]
Basically like returningId accept it should accept columns

**Comment by seratch** [comment]
Did you mean the `returningId` method in QueryDSL? If so, it's a quite simple thing. If you have time, we'd be appreciated to see your pull request to add the thing you need. https://github.com/scalikejdbc/scalikejdbc/blob/2.4.2/scalikejdbc-interpolation/src/main/scala/scalikejdbc/QueryDSLFeature.scala#L514

**Comment by kbrowder** [comment]
Yup, that's what I meant.  I may try to get on this in a couple of days.

**Comment by seratch** [comment]
Fixed by #564. Thank you for your contribution! 🎉

**Comment by jenwirth** [comment]
How is this suppose to work with a `where` clause. The `returning(...)` is a member of `UpdateSQLBuilder`, whereas a `where` returns a `ConditionSQLBuilder`

```scala
update(Post)
   .set(sqls"${p.views}=${p.views}+${newViews}")
   .where.eq(p.id,id)
   .returning(p.id,p.lastUpdated, p.views) // does not work as it is not a member of ConditionSQLBuilder
```
Thanks
